R/algorithm.R

```diff
@@ -98,7 +99,8 @@ create_SPC_auto_limits_table <- function(data,
       while(counter < nrow(data)){
         
         #[4]see whether there are enough points after the counter to form new period
-        if(enough_data_for_new_period(limits_table, periodMin, counter)){
+        if(enough_data_for_new_period(limits_table, periodMin, counter,
+                                      chartType = chartType)){
```

Comment:
@HorridTom this is not compliant with the tidyverse style guide! Suggest inserting newlines here to comply with the guide.